common/src/main/java/com/wynntils/mc/mixin/ItemStackMixin.java

```diff
@@ -26,4 +32,14 @@ private Component redirectGetHoveredName(ItemStack instance) {
         ItemTooltipHoveredNameEvent event = EventFactory.onGetHoverName(instance.getHoverName(), instance);
         return event.getHoveredName();
     }
+
+    @Override
+    public ItemAnnotation getAnnotation() {
```

Comment:
Do these "need" to be made `@Unique` too?

Reply:
I can't I fully understand the sponge documentation for the `@Unique` annotation, but I guess it does not hurt. Bit worried that they claim that they will "rename" it, how will method resolution work in that case?